src/altimate.ts

```diff
@@ -492,12 +493,24 @@ export class AltimateRequest {
     return response;
   }
 
+  private throwIfLocalMode(endpoint: string) {
+    const isLocalMode = workspace
+      .getConfiguration("dbt")
+      .get<boolean>("isLocalMode", false);
+
+    if (!["auth_health"].includes(endpoint) && isLocalMode) {
```

Comment:
@saravmajestic , @suryaiyer95 , @anandgupta42 : we need to list all endpoints that should be permitted. For example defer to prod configurations or check configurations, user list, ...